sleap/gui/commands.py

```diff
@@ -2693,6 +2698,23 @@ def ask_and_do(context: CommandContext, params: dict):
         if DeleteDialog(context).exec_():
             context.signal_update([UpdateTopic.project_instances])
 
+class AddInstanceGroup(EditCommand):
+    @staticmethod
+    def do_action(context, params):
+        frame_idx = context.state["frame_idx"]
+        session = context.state["session"]
+
+        if frame_idx in session.frame_groups:
+            frame_group = session.frame_groups[frame_idx]
+        else:
+            print(f"No frame group found for frame index {frame_idx}. Unable to add instance group.")
+            return
```

Comment:
This will actually also be the GUI menu for creating `FrameGroup`s if they do not exist.
```suggestion
        frame_group = session.frame_groups.get(frame_idx, None) or session.new_frame_group(frame_idx=frame_idx)
```